src/main/java/com/rtb/projectmanagementtool/auth/AuthOps.java

```diff
@@ -27,8 +27,22 @@ public AuthOps(DatastoreService ds) {
     UserController controller = new UserController(ds);
   }
 
-  // generates cookie. Used in most methods.
-  public Cookie generateCurrCookie(HttpServletRequest request, HttpServletResponse response) {
+  public String getAuthID() {
+    // call auth service to generate AuthID
+    UserService userService = UserServiceFactory.getUserService();
+    String AuthID = userService.getCurrentUser().getUserId();
+    return AuthID;
+  }
+
+  // get/create cookie and set value to userID
+  public void createAndSetCookieNewUser(HttpServletRequest request, long userID) {
```

Comment:
I'm not sure I understand this method, since Cookie here doesn't go anywhere.  If you intended to send this cookie back to the user's browser, you'll then need to do response.addCookie